packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -243,6 +245,11 @@ class FloatingActionButton extends StatelessWidget {
   /// Defaults to [ThemeData.hoverColor] for the current theme.
   final Color hoverColor;
 
+  /// The splash color for this [FloatingActionButton]'s [InkWell].
+  ///
+  /// Defaults to [ThemeData.splashColor] for the current theme.
```

Comment:
I know the other properties don't say this, but should this say that it fallsback to the FAB theme data and then the Themedata.splashcolor?

And the other properties should probably be updated to say this too

Reply:
Done.